examples/table_of_integrals.py

```diff
@@ -17,33 +17,64 @@
 seed(2021)
 rcParams['image.aspect'] = 'auto' # needed for sphinx gallery
 
-# sphinx_gallery_thumbnail_number = 1
+# sphinx_gallery_thumbnail_number = 7
```

Comment:
when I run this, I don't get the nice image with subplots that we had gotten previously -- is the issue that you have the code to do that with the old-style image plot, but not with DCCT?